### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#328] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-21 105414](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f298049d-37c1-4778-bbc2-66ffa2f4d6a6)

**Comment by Elia-Belli** [comment]
- $\mathbb{R}_3[x]\leq \mathbb{R}[x]$, infatti la somma di due polinomi di grado $\leq 3$ genera sempre un polinomio di grado $\leq 3$ perchè ad essere sommati sono solamente i coefficienti, stesso vale nel prodotto per uno scalare che va a modificare solamente il valore dei coefficiente lasciando invariato il grado del polinomio.
- $W=\\{insieme\ dei\ polinomi\ di\ grado\ \leq 3\ con\ soluzione\ 1)$
   - Siano $ax^3+bx^2+cx+d,a'x^3+b'x^2+c'x+d'\in W\Rightarrow (a+b+c+d)=0,\ (a'+b'+c'+d')=0$
ci chiediamo se la loro somma $s(x)=ax^3+bx^2+cx+d+a'x^3+b'x^2+c'x+d'\in W$, ovvero se $s(1)=0$
$s(1)=(a+b+c+d)+(a'+b'+c'+d')=0+0=0\Rightarrow s(x)\in W$
   - Controlliamo anche che il prodotto per uno scalare sia ancora in $W$:
Sia $p(x)=\alpha(ax^3+bx^2+cx+d)=\alpha ax^3+\alpha bx^2+\alpha cx+\alpha d$
$p(1)=\alpha a+\alpha b+\alpha c+\alpha d=\alpha (a+b+c+d)=\alpha \cdot 0=0 \Rightarrow p(x)\in W$
$\Rightarrow W\leq \mathbb{R}_3[x]$
- $U=\\{q(x)\in \mathbb{R}_3[x]|q(2)=4\\}$
Siano $ax^3+bx^2+cx+d,a'x^3+b'x^2+c'x+d'\in U\Rightarrow (8a+4b+2c+d)=4,\ (8a'+4b'+2c'+d')=4$
Controlliamo se la loro somma è ancora in $U$:
$s(x)=ax^3+bx^2+cx+d+a'x^3+b'x^2+c'x+d'$
$s(2)=(8a+4b+2c+d)+(8a'+4b'+2c'+d')=4+4\neq 4\Rightarrow s(x)\notin U$
$\Rightarrow U$ non è un sottospazio di $\mathbb{R}_3[x]$

**Comment by daromacarlo** [comment]
![Immagine 2023-11-21 215703](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b25cfe02-99b0-467e-a508-4474959514d1)
![Immagine 2023-11-21 215733](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/ae937ba6-0be2-4275-9e39-6a185414fb56)
![Immagine 2023-11-21 215804](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/aa35a0d4-0a19-47df-ac6e-d5384a79acb7)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![Screenshot 2023-11-29 112519](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cc785c31-1dad-431c-90ac-5ba003634d0d)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/0c0d99a1-db76-45d8-beb2-c4df03ee9073)
